rpc/pb/wrappers/pb-marshalling.go

```diff
@@ -0,0 +1,263 @@
+// Copyright 2016 ISRG.  All rights reserved
```

Comment:
This and below should be in `grpc/` since it is for a completely different RPC layer than the files in `rpc` this also means once we delete all the AMQP code we won't have to merge the `grpc` and `rpc` folders.